index.html

```diff
@@ -53,10 +54,55 @@
 
   <title>CALCULATOR | YOSHITHA</title>
 
+  <style>
+    html{
+    scroll-behavior: smooth;
+}
+/* Scroll back to top styles */
+
+.scrolltop{
+    position: fixed;
+    left: 1rem;
+    bottom: -20%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: .5rem;
+    background: rgba(0, 0, 0, 0.5);
+    border-radius: .4rem;
+    z-index: 10;
+    transition: .4s;
+    text-decoration: none;
+    visibility: hidden;
+}
+
+.scrolltop:hover{
+    background-color: #26eab9af;
+}
+
+.scrolltop__icon{
+    font-size: 1.8rem;
+    color: #fff;
+}
+
+.scroll-top{
+    visibility: visible;
+    bottom: 1.5rem;
+}
+
+  </style>
+
 </head>
 
 <body class="bg-img">
+
+
+  <!-- Scroll back to top icon -->
```

Comment:
Can you add these scroll to top button code after the navbar @Devemmy01

Reply:
All changes has been made @YoshithaRathnayake